File: Codigo/2 - Ad-Hoc/src/3465.py
"""
Created on 2025-03-20 at 03:32:27

@author: GabrielCampelo
"""

import math

def main():
    a, b, c = map(int, input().split())

    p = (a + b + c) / 2.0

    area = math.sqrt(p * (p - a) * (p - b) * (p - c))

    print(f"{area:.3f} m2")

if __name__ == '__main__':
    main()

File: Codigo/2 - Ad-Hoc/src/3465.c
/**
* @file 3465.c
*
* Created on 2025-03-20 at 03:30:13
* @author GabrielCampelo
*/

#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <stdbool.h>
#include <math.h>

#define min(a, b) ((a) < (b) ? (a) : (b))
#define max(a, b) ((a) > (b) ? (a) : (b))

#define INF 0x3f3f3f3f
#define LINF 0x3f3f3f3f3f3f3f3fLL

typedef long long ll;
typedef unsigned long long ull;

int main() { 
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);

    double p = (a + b + c) / 2.0;

    double ans = p * (p - a) * (p - b) * (p - c);
    ans = sqrt(ans);

    printf("%.3lf m2\n", ans);
    
    return 0;
}